File: skills/astra-toolkit/clients/python/examples/client-get-db-admin-from-admin.py
import os
from astrapy import DataAPIClient

# Get an admin object
client = DataAPIClient(os.environ["ASTRA_DB_APPLICATION_TOKEN"])
admin = client.get_admin()

# Get a database admin object
database_admin = admin.get_database_admin(api_endpoint=os.environ["ASTRA_DB_API_ENDPOINT"])

File: skills/astra-toolkit/clients/typescript/examples/client-get-db-admin-from-admin.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get an admin object
const client = new DataAPIClient(process.env.ASTRA_DB_APPLICATION_TOKEN!);
const admin = client.admin();

// Get a database admin object
const databaseAdmin = admin.dbAdmin(process.env.ASTRA_DB_API_ENDPOINT!);
